### PR TITLE
document allowList for github auth

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Relevant slack discussion: https://okteto.slack.com/archives/C02T9BKMS4V/p1691047787338819

cc @omnomagonz @ifbyol

### CODE REVIEW

**Review by omnomagonz — APPROVED** [review]

File: src/content/self-hosted/administration/configuration.mdx [hunk 1]
```diff
@@ -157,7 +157,19 @@ auth:
...
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
+
+```yaml
+auth:
+  github:
+    enabled: true
+    clientId: clientID
+    clientSecret: clientSecret
+    organization: my-org
+    allowList = ["acct1","acct2","acct3"]
```
Comment (omnomagonz) [inline src/content/self-hosted/administration/configuration.mdx:171]:
```suggestion
    allowList: ["acct1","acct2","acct3"]
```

Is this the correct syntax for this line item? Noticed after hitting approve 🤦‍♂️

Comment (ifbyol) [reply src/content/self-hosted/administration/configuration.mdx:171]: Yes, your suggestion is the right syntax to use

File: src/content/self-hosted/administration/configuration.mdx [hunk 2]
```diff
@@ -157,7 +157,19 @@ auth:
 
 > You can also use a [secret](self-hosted/administration/configuration.mdx/#cloud-provider-secret) to store the sensitive part of these credentials.
 
-The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in.
+The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in. 
+
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
```
Comment (rberrelleza) [inline src/content/self-hosted/administration/configuration.mdx:162]: we should mention how organization and allowList work together. Or maybe we should pitch it as use one or the other?

**Comment by omnomagonz** [comment]
@RinkiyaKeDad is this PR good to merge or would you like to address Ramiro's comment above / any other parts of this you might be working on?

File: src/content/self-hosted/administration/configuration.mdx (hunk 2 above)
Comment (RinkiyaKeDad) [reply src/content/self-hosted/administration/configuration.mdx:162]: > Or maybe we should pitch it as use one or the other?

Is that what we want? I'm not sure on the intended use here. 

Do you think it's okay if we have these two fields documented in isolation (like right now) and let users connect the dots and decide if they want to use a combination of both or either one.

**Comment by omnomagonz** [comment]
@RinkiyaKeDad This PR now has merge conflicts. Would you like to resolve those or maybe open a new PR if that's easier? Would love to get this merged 👍

**Comment by RinkiyaKeDad** [comment]
Closing this in favour of https://github.com/okteto/docs/pull/588

**Pull request closed without merging**